cmd/nodecmd/stop.go

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	awsAPI "github.com/ava-labs/avalanche-cli/pkg/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+
+	"github.com/spf13/cobra"
+)
+
+func newStopCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stop [clusterName]",
+		Short: "(ALPHA Warning) Stop all nodes in a cluster",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
+
+The node stop command stops a running node in cloud server
+
+Note that a stopped node may still incur cloud server storage fees.`,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+		RunE:         stopNode,
+	}
+
+	return cmd
+}
+
+func removeNodeFromClusterConfig(clusterName string) error {
+	clusterConfig := models.ClusterConfig{}
+	var err error
+	if app.ClusterConfigExists() {
+		clusterConfig, err = app.LoadClusterConfig()
+		if err != nil {
+			return err
+		}
+	}
+	if clusterConfig.Clusters != nil {
+		delete(clusterConfig.Clusters, clusterName)
+	}
+	return app.WriteClusterConfigFile(&clusterConfig)
+}
+
+func removeDeletedNodeDirectory(clusterName string) error {
+	return os.RemoveAll(app.GetNodeInstanceDirPath(clusterName))
+}
+
+func removeClusterInventoryDir(clusterName string) error {
```

Comment:
this should be remove node from cluster inventory dir (similar in spirit to remove node from cluster config). can we add an issue for this? probably the inventory file should be regenerated based on the contents of the cluster config